packages/accounts-base/accounts_server.js

```diff
@@ -1433,15 +1449,22 @@ Ap.updateOrCreateUserFromExternalService = function (
     // We *don't* process options (eg, profile) for update, but we do replace
```

Comment:
This comment needs to change since we're now providing a way to process options. Please remove the `XXX` line, and consider updating this comment to better reflect the new functionality. Actually, you could consider just removing this comment entirely - now that we're addressing options, the code here pretty much speaks for itself.